recipes-kernel/linux/linux-riscv_4.20.bb

```diff
@@ -0,0 +1,8 @@
+require recipes-kernel/linux/linux-riscv-common.inc
```

Comment:
Can you also add `DEFAULT_PREFERENCE = "-1" `  for this please ?

Reply:
Fixed